polling_stations/apps/data_importers/base_importers.py

```diff
@@ -238,17 +238,11 @@ def check_station_point(self, station_record):
                 council = Council.objects.get(
                     geography__geography__covers=station_record["location"]
                 )
-                if self.council_id not in council.identifiers:
+                if self.council_id != council.council_id:
                     self.logger.log_message(
                         logging.WARNING,
-                        "Polling station %s is in %s (%s) but target council is %s (%s) - manual check recommended\n",
-                        variable=(
-                            station_record["internal_council_id"],
-                            council.name,
-                            council.council_id,
-                            self.council.name,
-                            self.council.council_id,
-                        ),
+                        f"Polling station {station_record['internal_council_id']} is in {council.name} ({council.council_id}) "
+                        f"but target council is {self.council.name} ({self.council.council_id}) - manual check recommended\n",
```

Comment:
Is there any reason for these to not be f-strings?

Reply:
Drive-by: The main reason why a lot of the code in this codebase uses `%` or `.format()` instead of f-strings is because a lot of it was written before python 3.6 came out (or before we were deploying on it), not because f-strings are bad. AFAIK you're on python 3.6 in prod (I think) so no reason not to start using them.